src/components/item.jsx

```diff
@@ -4,7 +4,7 @@ class Item extends React.Component {
   constructor(props) { //
     super(props);
     this.state = {
-      itemId: 1,  // from Tiers
+      itemId: 1,  // passed down via props
```

Comment:
So are you passing in props to this component when the instance of it is created when you call `ReactDOM.render()`?

Reply:
Correct, it's being passed along when the parent (Tier) generates this component. Then in componentDidMount, I'm assigning it into the state for this component.